File: src/main/ruby/hello_jena.rb
#!/usr/bin/ruby

require 'java'

java_import java.lang.System
java_import org.apache.jena.examples.Utils
java_import com.hp.hpl.jena.rdf.model.ModelFactory

input = Utils.getResourceAsStream("data/data.ttl")
model = ModelFactory.createDefaultModel()
model.read(input, nil, "TURTLE")

puts "\n---- Turtle ----"
model.write(System.out, "TURTLE")

File: src/main/scala/hello_jena.scala

import _root_.java.lang.System
import _root_.org.apache.jena.examples.Utils
import _root_.org.apache.jena.rdf.model.ModelFactory

object HelloJena {
    def main(args: Array[String]) {
        val input = Utils.getResourceAsStream("data/data.ttl")

        val model = ModelFactory.createDefaultModel()
        model.read(input, null, "TURTLE")

        System.out.println("\n---- Turtle ----")
        model.write(System.out, "TURTLE")
    }
}
